test/parity-spec.js

```diff
@@ -0,0 +1,55 @@
+import { describeWithDOM } from './_helpers';
```

Comment:
I think this is an important thing to test for any API method that should have feature parity between `mount` and `shallow` 👍